worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -502,7 +504,9 @@ class PushDataHandler extends BaseMessageHandler with Logging {
         } else {
           StatusCode.PUSH_DATA_FAIL_SLAVE.getMessage()
         }
-      callbackWithTimer.onFailure(new Exception(s"$message! ${locations.head}", exception))
+      callbackWithTimer.onFailure(new Exception(
+        s"$message! ${partitionIdToLocations.head}",
```

Comment:
does it need to show tuple or just partitionIdToLocations.head._2 here?

Reply:
Done